packit_service/worker/jobs.py

```diff
@@ -66,6 +67,21 @@ def do_we_process_fedmsg_topic(topic: str) -> bool:
     return topic in PROCESSED_FEDMSG_TOPICS
 
 
+class Status(Enum):
+    success = "success"
+    failure = "failure"
+
+
+class HandlerResults:
+    def __init__(self, status: Status, details: dict = None):
+        super().__init__()
+        self.status = status
+        self.details = details or {}
```

Comment:
What structure should have this dict? I can see that in case of a failure you are storing `details={"error": {"msg": msg}`. This could be documented here.

Reply:
commented